Documentation/authentication.md

```diff
@@ -1,5 +1,7 @@
 # Authentication Guide
 
+**NOTE: The authentication feature is considered as an experimental. We might change workflow without warning in the future releases.**
```

Comment:
**NOTE: The authentication feature is considered experimental. We may change workflow without warning in future releases.**